topology/probes/k8s/cache.go

```diff
@@ -211,6 +212,10 @@ func (c *ResourceCache) OnAdd(obj interface{}) {
 	defer c.graph.Unlock()
 
 	id, metadata := c.handler.Map(obj)
+	if len(c.clusterName) > 0 {
+		metadata.SetFieldAndNormalize("ClusterName", c.clusterName)
```

Comment:
SetField() is enough (no normalization needed)

Reply:
should be ClusterName not "ClusterName"

Reply:
and better even to rename to ClusterNameField

Reply:
Done